x/auction/keeper/auctions.go

```diff
@@ -443,7 +449,7 @@ func (k Keeper) PlaceBidDebt(ctx sdk.Context, auction *types.DebtAuction, bidder
 		auction.MaxEndTime = ctx.BlockTime().Add(k.GetParams(ctx).MaxAuctionDuration) // set maximum ending time on receipt of first bid
 		auction.HasReceivedBids = true
 	}
-	auction.EndTime = earliestTime(ctx.BlockTime().Add(k.GetParams(ctx).BidDuration), auction.MaxEndTime) // increment timeout, up to MaxEndTime
+	auction.EndTime = earliestTime(ctx.BlockTime().Add(k.GetParams(ctx).ReverseBidDuration), auction.MaxEndTime) // increment timeout, up to MaxEndTime
```

Comment:
I think it would be safer to use the longer forward bid duration here. These auctions trigger when collateral auctions have failed to raise enough usdx to cover debt. So the system mints kava and sells it to raise more usdx. If these auctions clear too fast, a huge amount of kava could be minted.